target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -101,6 +101,14 @@ engenius,ecb1750)
 devolo,magic-2-wifi)
 	ucidef_set_led_netdev "plcw" "dLAN" "devolo:white:dlan" "eth0.1" "rx"
 	;;
+dlink,dap-1330-a1|\
+dlink,dap-1365-a1)
+	ucidef_set_rssimon "wlan0" "2000000" "2"
+	ucidef_set_led_rssi "wifi-low" "wifi-low" "d-link:red:wifi" "wlan0" "1" "29"
+	ucidef_set_led_rssi "wifi-medium" "wifi-medium" "d-link:green:wifi" "wlan0" "30" "100"
+	ucidef_set_led_rssi "wifi-high" "wifi-high" "d-link:green:signal1" "wlan0" "50" "100"
+	ucidef_set_led_rssi "wifi-max" "wifi-max" "d-link:green:signal2" "wlan0" "70" "100"
```

Comment:
I see that you copied this from ar71xx. But if you ask me, this is nonsense:
- 2000000 should be 200000 (and is it really 2 for the last parameter)
- wifi_red/wifi_green look to me like the should be status indicators
- wifi-low parameters don't make sense

Please research what behavior these LEDs should have based on vendor behavior.

Reply:
The RSSI graph consists of three fields, only the lower one being composed of two LEDs red and green. So I would assume this would change from green to red when the signal is very low (which I honestly could not reproduce even when using tin foil, is there any way the RSSI value could be mocked?). I'll check the rssileds package again and dig into the vendor behaviour.

Reply:
Ah, I got it now. So, the arguments for the ucidef_set_led_rssi lines are actually correct that way, as the red led will turn off after 29 % and the "medium" one will only start to light at 30 %. So, the only thing to change would be the names, and check the rssimon parameters, of course.

Reply:
Despite, I wouldn't use too much time on testing that. I'd just rename the LEDs and use the standard parameters for rssimon ...